### DISCUSSION TITLE
How can I download bottles/binaries now that bintray is gone?

### DISCUSSION THREAD

**Discussion by shrinidhi111** [Homebrew/discussions#1926] [Everyday usage]
I have been digging around have learnt that all the binaries have been moved to GitHub registry as OCI images.

Before I could download a "bottle" or binary manually, but I cannot for the life of me figure out how to convert the docker images to the older binaries form...

I am specifically interested in [wireguard](https://github.com/orgs/Homebrew/packages?tab=packages&q=wireguard) and their dependency - bash 5.

**Comment by SMillerDev** [comment]
This should allow you to download tools: https://github.com/containers/oci-fetch

**Comment by alebcay** [comment]
If you already have `brew` installed, you can also still `brew fetch <formula>` and get a plain tarball of the bottle just like before.
